ecs-init/config/awsrulesfn/partitions.go

```diff
@@ -0,0 +1,403 @@
+// Code generated by endpoint/awsrulesfn/internal/partition. DO NOT EDIT.
```

Comment:
[This `go generate`](https://github.com/aws/amazon-ecs-agent/blob/85a28b923aeb9a6a61e818c8cf4d34892fd49f83/ecs-init/config/awsrulesfn/partitions.gen.go) copies [this AWS SDK file](https://github.com/aws/aws-sdk-go-v2/blob/main/internal/endpoints/awsrulesfn/partitions.go) from vendor into our package.